core/src/main/java/org/apache/accumulo/core/client/impl/DelegationToken.java

```diff
@@ -14,7 +14,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.apache.accumulo.core.client.security.tokens;
+package org.apache.accumulo.core.client.impl;
```

Comment:
I will try to create an opaque DelegationToken type.  It seems like that gives us more API flexibility in the future, even if we don't have any extra public methods now.